### DISCUSSION TITLE
Rust Programlama Dili Türkçe Çevirisi -2021

### DISCUSSION THREAD

**Discussion by RustDili** [RustDili/rust-book-tr#2] [Duyurular]
## 👋 Hoş geldiniz!
  Rust progamlama dili öğreticisinin 2021 sürümü Türkçe çevirisi hakkında  soru fikir ve önerilerimizi burada paylaşabiliriz:

**Comment by setanarut** [comment]
Merhaba iki ayrı konuda önerim olacak ilk izlenimim bu şekilde oldu.

## Repo karmaşası

Aynı dökümanları içeren repolar birbirine bağlantı veriyor ve açıklamalar birbirini tutmuyor örneğin https://github.com/RustDili açıklaması "Türkçe kaynak havuzu" içindeki  dökümanlar reposunda "Dili Bilgi-Belge Havuzu" yazıyor. Yani `Türkçe kaynak havuzu > Rust Programlama Dili Bilgi-Belge Havuzu` şeklinde hiyerarşi sorunu var.
Tam bir karmaşa hakim. Bir reponun Readme dosyasında başka repolara bağlantılar verilmiş (https://github.com/RustDili/dokuman#readme)
Bence aktif olmayan çeviri repoları arşivlenmeli ve tüm kitaplar tek bir repoda sade bir şekilde depolanmalı. Böylece katılımcılar ayrı ayrı repolarda kaybolmadan her kitap çevirisi için aynı repodan çekme isteği gönderebilir ve tek bir içindekiler sayfasından tüm kitaplara ulaşabilir.

## Repo açıklaması

Repo açıklaması hatalı "🇹🇷 Rust Programlama Dili Türkçe Çevirisi" sanki dilin kendisi çevrilmiş gibi anlaşılıyor. Kitabın orjinal dilindeki ismi yazılmalıydı. Şu şekilde;

> The Rust Programming Language Türkçe Çevirisi

ya da

> The Rust Programming Language kitabının Türkçe Çevirisi

**Reply by RustDili** [reply]
Merhaba,

> The Rust Programming Language kitabının Türkçe Çevirisi

Önerileriniz için teşekkür ederim. İsimlendirme hatasını düzelttim. Repo karmaşasını da yakın zamanda düzeltelim.
